python_modules/libraries/dagster-databricks/dagster_databricks/databricks_pyspark_step_launcher.py

```diff
@@ -31,6 +35,7 @@
         'databricks_token': Field(
             StringSource, is_required=True, description='Databricks access token',
         ),
+        'secrets': define_databricks_secrets_config(),
```

Comment:
I think it's worth being a little more explicit in the name about the weird thing that's happening here, e.g. `secrets_to_env_variables` or something.

Reply:
Fixed.